setup.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+
+def get_requirements(fn):
+    with open(fn, "r") as fp:
+        return [req.strip() for req in fp.readlines()]
+
+
+setup(
+    name='dockhub',
+    author='ckolos',
+    version='1.0.0',
+    url='https://github.com/ckolos/dockhub.py/',
+    install_requires=get_requirements("requirements.txt"),
+    license='MPLv2',
+    entry_points="""
+    [console_scripts]
+    dockhub=dockhub:main
+    """,
```

Comment:
`entry_points` does some magic on installation where it'll take the `dockhub.py` script, wrap it in a wrapper, and make it an "installed script" in the environment.

I couldn't find any explanation for it or the syntax, so ... yay.